pkg/ccl/backupccl/backup_destination.go

```diff
@@ -344,13 +344,12 @@ func readLatestFile(
 func findLatestFile(
 	ctx context.Context, exportStore cloud.ExternalStorage,
 ) (ioctx.ReadCloserCtx, error) {
-	latestFile, err := exportStore.ReadFile(ctx, latestHistoryDirectory+"/"+latestFileName)
+	latestFile, err := exportStore.ReadFile(ctx, backupMetadataDirectory+"/"+latestHistoryDirectory+"/"+latestFileName)
 	if err != nil {
-		if !errors.Is(err, cloud.ErrFileDoesNotExist) {
-			return nil, err
-		}
-
 		latestFile, err = exportStore.ReadFile(ctx, latestFileName)
+		if err != nil {
+			return nil, errors.Wrap(err, "Latest file could not be read in base or metadata directory")
```

Comment:
nit: s/Latest/LATEST/g

Reply:
Wasn't 100% sure what you meant by this, did you mean in place of "latest file" or the "metadata directory"

Reply:
ahh just change `Latest` to `LATEST` since thats the sentinel file name we use.